connector_dns/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+=============
+Connector Dns
+=============
+
+This module aims to allows to manage your DNS domain through Odoo.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (connector)
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+ * No specific configuration needed.
+
+Usage
+=====
+
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Eric Caudal <eric.caudal@elico-corp.com>
+* Noah Wang <noah.wang@elico-corp.com>
+* Liu Lixia <liu.lixia@elico-corp.com>
+* Augustin Cisterne-Kaas
```

Comment:
add email address: contact@elico-corp.com and change the one from lixia to the same address